app/views/events/index.html.erb

```diff
@@ -1,31 +1,27 @@
-<h1>Events</h1>
-
-<table>
-  <thead>
-    <tr>
-      <th>Name</th>
-      <th>Duration</th>
-      <th>Started at</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
+<div class="container">
+  <table class= "table table-hover">
+    <thead>
+      <tr>
+<div class="container">
+  <h1> <%= t('.events') %> </h1>
   <tbody>
-    <% @events.each do |event| %>
+  <table class= "table table-hover">
+    <thead>
+          <% @events.each do |event| %>
```

Comment:
It looks for me that there is a bug here. Did you check if you see all events on this page?

Reply:
<tbody>
    <% @events.each do |event| %>
    <table class= "table table-hover">
      <thead>
        <tr>
          <th> <%= t('.name_event') %> </th>
          <th><%= t('.duration') %> </th>
          <th><%= t('.start_at') %> </th>
          <th colspan="3"></th>
        </tr>
      </thead>
    </table>
    <% end %>
  </tbody>

Is the correct change? 
Now, it doesn`s show anything... :(

Reply:
No, this is exactly the problem here. In lines 16-21 you removed all the data about event and you put name of the column in the wrong place.